purchase_request/README.rst

```diff
@@ -1,7 +1,3 @@
-.. image:: https://odoo-community.org/readme-banner-image
```

Comment:
@sbidoul look at this change. The template in the pre-commit definition is not the same as the one being applied "on ocabot" part, provoking this eternal removal and introduction each time a manifest is touched. IMO, ocabot should execute the pre-commit version of the repository, not one fixed.

Reply:
This is the result of a compromise between many different requirements. If you want to change this, please open an issue on https://github.com/OCA/oca-addons-repo-template/, but before that please spend sometime re-reading all the arguments that were discussed back then.